source/_integrations/frontier_silicon.markdown

```diff
@@ -8,58 +8,38 @@ ha_release: '0.40'
 ha_domain: frontier_silicon
 ha_platforms:
   - media_player
+ha_codeowners:
+  - '@wlcrs'
+ha_ssdp: true
+ha_config_flow: true
 ---
 
 This integration provides support for Internet Radios based on the [Frontier Silicon chipset]. Some of the manufacturers which offer products based on these chips include: Hama, Medion, Slivercrest, Auna, Technisat, Revo, Pinnel, etc. These devices will be usually controlled by the [UNDOK] app.
 
+{% include integrations/config_flow.md %}
```

Comment:
I think this makes more sense to add after the prerequisites?

Reply:
So maybe you misunderstood me? Thus unresolving.

I think the order should be:

- Supported models
- Prerequiresites
- Configuration

Reply:
Should be ok now, sorry!